cli/network.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/spf13/cobra"
+)
+
+// networkDescription defines the network command description and auto generate command doc.
+var networkDescription = "Manager the networks in pouchd. " +
+	"It contains the functions of create/remove/list/inspect network, 'driver' is used to list drivers that pouch support. " +
+	"Now bridge network is supported in pouchd defaulted, it will be initialized when pouchd starting."
+
+// NetworkCommand is used to implement 'network' command.
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
+		Long:  networkDescription,
+		Args:  cobra.MinimumNArgs(1),
+	}
+
+	c.AddCommand(n, &NetworkCreateCommand{})
+}
+
+// networkCreateDescription defines the comand network create description.
+var networkCreateDescription = "Create a network in pouchd. " +
+	"It must specify network's name and driver. You can use 'network driver' to get drivers that pouch support."
+
+// NetworkCreateCommand is used to implement 'network create' command.
+type NetworkCreateCommand struct {
+	baseCommand
+
+	name       string
+	driver     string
+	gateway    string
+	ipRange    string
+	ipamDriver string
+	ipamOpts   []string
+	subnet     []string
+	options    []string
+	labels     []string
+}
+
+// Init initializes NetworkCreateCommand command.
+func (n *NetworkCreateCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create a pouch network",
+		Long:  networkCreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return n.runNetworkCreate(args)
+		},
+		Example: networkCreateExample(),
+		Args:    cobra.MaximumNArgs(1),
+	}
+
+	n.addFlags()
+}
+
+// addFlags adds flags for specific command.
+func (n *NetworkCreateCommand) addFlags() {
+	flagSet := n.cmd.Flags()
+
+	flagSet.StringVarP(
+		&n.name,
+		"name",
+		"n",
+		"",
+		"the name of network")
```

Comment:
Similarly, I think put all these fields in one line which is in consistent with other files' behavior is much better.

Reply:
ok